fastlane/lib/fastlane/actions/swiftlint.rb

```diff
@@ -89,7 +102,7 @@ def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :mode,
                                        env_name: "FL_SWIFTLINT_MODE",
-                                       description: "SwiftLint mode: :lint, :autocorrect or :analyze",
+                                       description: "SwiftLint mode: :lint, :fix, :autocorrect or :analyze",
                                        is_string: false,
```

Comment:
You can use `type: Symbol` here and delete `is_string`

Reply:
Thank you for the review, applied suggestion! 😇